resources/assets/js/management/roles/components/RolesListing.vue

```diff
@@ -11,10 +11,42 @@
               <b-btn variant="action" @click="onAction('permissions-item', props.rowData, props.rowIndex)" v-b-tooltip.hover title="Permissions"><i class="fas fa-user-lock"></i></b-btn>
             </div>
           </div>
-      </template>  
-    </vuetable> 
+      </template>
+    </vuetable>
     <pagination single="Role" plural="Roles" :perPageSelectEnabled="true" @changePerPage="changePerPage" @vuetable-pagination:change-page="onPageChange" ref="pagination"></pagination>
-   </div>
+    <b-modal ref="editItem" size="md" centered title="Create New Role">
+    <form>
+      <div class="form-group">
+        <label for="add-role-code">Code</label>
+        <input id="add-role-code" class="form-control" v-model="code">
+      </div>
+      <div class="form-group">
+        <label for="add-role-name">Name</label>
+        <input id="add-role-name" class="form-control" v-model="name">
+      </div>
+      <div class="form-group">
+        <label for="add-role-name">Description</label>
+        <input id="add-role-name" class="form-control" v-model="description">
+      </div>
+      <div class="form-group">
+        <label for="add-role-status">Status</label>
+        <select class="form-control" id="add-role-status" v-model="status">
+          <option :value="status">{{status}}</option>
+          <option value="ACTIVE">Active</option>
+          <option value="DISABLED">Disabled</option>
+        </select>
+      </div>
+    </form>
+    <template slot="modal-footer">
+      <b-button @click="hideEditModal" class="btn-outline-secondary btn-md">
```

Comment:
Can we use the built-in methods to hide the modal?

Reply:
I am using the built in hide() method from vuebootstraps documentation, Is there another way that I missed?